pkg/sdn/plugin/common.go

```diff
@@ -148,7 +196,7 @@ func getNetworkInfo(osClient *osclient.Client) (*NetworkInfo, error) {
 		return nil, err
 	}
 
-	return parseNetworkInfo(cn.Network, cn.ServiceNetwork)
+	return parseNetworkInfo(cn.ClusterNetworks, cn.ServiceNetwork)
```

Comment:
I don't think, we need to check overlap among cluster network cidrs here. 
As commented before, I believe this could be done in ValidateClusterNetwork().